pom.xml

```diff
@@ -267,6 +267,37 @@
           </systemProperties>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4.2.jenkins-1</version>
+        <configuration>
+          <rules combine.children="append">
+            <requireUpperBoundDeps>
+              <excludes>
+                <!-- TODO Stapler requests 14 while Jenkins requests 11; should backdate the Stapler version -->
+                <exclude>com.google.guava:guava</exclude>
```

Comment:
This is defined in the parent. You should not repeat it here. `combine.children` works when you actually put it in the right place—on `excludes`.